### DISCUSSION TITLE
baresip -Webrtc first frame got stuck with actual video from device node.

### DISCUSSION THREAD

**Discussion by Veeramanikandan14** [baresip/baresip#3036] [Q&A]
Hello there, 

I was testing the baresip webrtc module, with test source it worked successfully. i could be able to see the test video streaming after the sip negotiation. In the source code i made the below changes, its reads from the camera device, but got stuck after the first frame. 

        //str_ncpy(config->video.src_dev, "lavfi,testsrc2",
        str_ncpy(config->video.src_dev, "/dev/video0",
                 sizeof(config->video.src_dev));
Pls, suggest any other changes need to do. 

Regards,
Balaji T

**Comment by alfredh** [comment]
please include more info:

https://github.com/baresip/baresip/wiki/Guidelines-for-reporting-Issues
